js/main.js

```diff
@@ -16,4 +15,108 @@ function getRandomFloat (min,max, count) {
     return parseFloat(random);
   } return NaN;
 }
-getRandomFloat();
+
+const OFFERSCOUNT = 10;
+
+const types = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+
+const price = {
+  min: 1000,
+  max: 100000,
+};
```

Comment:
```suggestion
const Price = {
  MIN: 1000,
  MAX: 100000,
};
```

https://up.htmlacademy.ru/javascript/27/criteries#b8